agent/src/main/scala/io/scalac/mesmer/agent/akka/actor/impl/ActorMetricsInitAdvice.scala

```diff
@@ -0,0 +1,46 @@
+package io.scalac.mesmer.agent.akka.actor.impl
+
+import akka.actor.ActorContext
+import com.typesafe.config.ConfigFactory
+import net.bytebuddy.asm.Advice
+import net.bytebuddy.asm.Advice.OnMethodEnter
+
+import io.scalac.mesmer.agent.akka.actor.AkkaActorAgent
+import io.scalac.mesmer.core.actor.ActorCellDecorator
+import io.scalac.mesmer.core.actor.ActorCellMetrics
+
+object ActorMetricsInitAdvice {
+
+  @OnMethodEnter
+  def initializeMetrics(@Advice.This cell: ActorContext): Unit = {
+
+    val metrics = new ActorCellMetrics()
+    val config  = AkkaActorAgent.module.enabled(ConfigFactory.load())
```

Comment:
We could use Advice.withCustomProperties().bind()`. Alternatively we can wrap the config in an object that loads it once and use a getter here to avoid loading the config multiple times. 

I will see what's best here